pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/editor/MutationsAnnotator.kt

```diff
@@ -41,6 +41,17 @@ class MutationsAnnotator :
             return mutationResults.mapIndexed { index, it -> "${index + 1}. ${it.description} → ${it.status}" }
                 .joinToString(separator)
         }
+
+        fun formatTooltipMessage(message: String): String {
+            return """
+                <html>
+                     <body style="font-family: 'Arial', sans-serif; font-size: 12px;">
```

Comment:
I think we should adhere to the users font which can come out of `UIUtil.getToolTipFont()` for example